documentation/modules/ROOT/pages/03-container-image-security.adoc

```diff
@@ -23,7 +23,7 @@ Project Thoth offers container https://developers.redhat.com/articles/2021/11/25
 * Container image metadata as extracted by https://github.com/containers/skopeo[Skopeo]
 * Information about other libraries, such as the CUDA version (GPU software) available
 
-All these data can give you insights on the software that is available for use.
+All these data can give you insights on the software that is available for use. Together with static analysis of container images as performed by `Quay Clair <https://github.com/quay/clair>`__, Thoth guides also on the base container image security.
```

Comment:
```suggestion
All these data can give you insights on the software that is available for use. Together with static analysis of container images as performed by `Quay Clair <https://github.com/quay/clair>`__, 
Thoth guides also on the base container image security.
```

Not sure though but I think this might cause issues otherwise

Reply:
You mean the link? Sorry, I must have used reStructuredText syntax. Will fix, thanks for spotting this 👍🏻

Reply:
Yes, thank you! :100: